pkg/swagger/swagger.go

```diff
@@ -89,12 +89,30 @@ func Run(api, outputDir string) error {
 		},
 	}
 
+	// for installaOpenShiftVerions
```

Comment:
This comment serves no real purpose and should be expanded or removed IMO.

Reply:
Looking at the comment after the change, and @s-amann's comment, this does need to be removed.

Reply:
Done